demo-project/src/demo_project/pipelines/reporting/nodes.py

```diff
@@ -90,6 +95,36 @@ def make_price_analysis_image(model_input_table: pd.DataFrame) -> PIL.Image:
     return pil_table.image
 
 
+def create_feature_importance_plot(features: pd.DataFrame):
+    sorted_features = features.sort_values(by="Score", ascending=True)
+    plotly_object = px.bar(
+        sorted_features,
+        x="Score",
+        y="Features",
+        color="Score",
+        color_continuous_scale="RdBu",
+    )
+    return plotly_object
+
+
 def create_matplotlib_chart(companies: pd.DataFrame):
-    plt.plot([1, 2, 3], [4, 5, 6])
+    random_actuals = []
+    random_predicted = []
+    for i in range(0, 11):
+        actual = random.randint(0, 1)
+        predicted = random.randint(0, 1)
+        random_actuals.append(actual)
+        random_predicted.append(predicted)
+
+    data = {"y_Actual": random_actuals, "y_Predicted": random_predicted}
+    plt.style.use("dark_background")
```

Comment:
Is this because it looks nicer on the website? It's pretty minor but I found the color scheme a bit unnatural

Reply:
yes, it is just so that it looks good on Kedro-viz dark theme